libs/ui/src/smart_cards_screen.tsx

```diff
@@ -59,6 +59,7 @@ const CardActions = styled.div`
   padding: 1rem;
   display: flex;
   flex-direction: column;
+  flex-grow: 1;
```

Comment:
This just makes sure the right panel is always full width to prevent flicker like so - notice the error message go from being wider to narrower:

https://github.com/user-attachments/assets/d11ec594-7741-4b0b-a617-2ef4741c4cfd